hdl21/primitives.py

```diff
@@ -531,7 +558,7 @@ class SineVoltageSourceParams:
     phase = Param(dtype=Optional[Scalar], default=None, desc="Phase at td (degrees)")
 
 
-_add(
+SineVoltageSource = _add(
```

Comment:
These assignments are helping with IDE references, right? 

Assuming so, a good thing to have. 
Some history: that `_add` method got "added" when one or more of them got outta sync with its `name` field, and that confused export/ import steps. 
With this as-is, `_add` is setting a module-level `SineVoltageSource` attribute, this assignment is over-riding it, and even if we, say, spelled this assignment `SignVoltageSource` (i.e. a typo), both should be available.